src/web/site_admin.ml

```diff
@@ -2406,14 +2406,14 @@ let () =
            | None ->
               return (
                   Printf.ksprintf Ocsigen_messages.warning
-                    "Unsuccessful attempt to change the password of %S (%S) for service %s"
+                    "Could not change the password of %S (%S) for the %s service"
                     username email service
                 )
            | Some (username, address) ->
               Web_signup.send_changepw_link ~service ~address ~username
          in
          let message =
-           s_ "If the account exists, an e-mail was sent with a confirmation link. Please click on it to change your password."
+           s_ "If the account exists, an e-mail containing a confirmation link will be sent. Please click it to change your password."
          in
```

Comment:
Why did you change the tense?

Reply:
Because of the former part, and also it should have been something like "if the account was found to exist, an e-mail would have been sent" or something, but that is complex.